packages/indexer-agent/src/commands/start.ts

```diff
@@ -52,6 +52,33 @@ export default {
         default: 4000,
         group: 'Ethereum',
       })
+      .option('gas-bump-time-limit', {
+        description:
+          'Time limit for transactions after which the transaction will be resubmitted with more gas (s)',
+        type: 'number',
+        default: 240,
+        group: 'Ethereum',
+        coerce: arg => arg * 1000,
+      })
+      .option('gas-bump-percent', {
+        description:
+          'The percent to bump the gas price before resubmitting a stuck transaction (%)',
+        type: 'number',
+        default: 20,
+        group: 'Ethereum',
+      })
+      .option('gas-price-max', {
+        description: 'The maximum gas price to use for a transaction (gwei)',
+        type: 'number',
+        default: 2000000000,
+        group: 'Ethereum',
+      })
+      .option('max-retries', {
```

Comment:
Is this the number of retries or the number of attempts? (There's a difference of 1 between them)

I'd call this `transaction-retries` or `transaction-attempts`?

Reply:
Attempts. Good point,  updating the name